feature-libs/product-configurator/rulebased/components/attribute/types/radio-button/configurator-attribute-radio-button.component.ts

```diff
@@ -12,6 +13,7 @@ export class ConfiguratorAttributeRadioButtonComponent
   extends ConfiguratorAttributeSingleSelectionBaseComponent
   implements OnInit
 {
+  uiType = Configurator.UiType;
```

Comment:
Is it necessary to add a UI type here?
`validationType` could be used to differentiate between numeric and alphanumeric

Reply:
See conversation above...